test/e2e/common/node/podtemplates.go

```diff
@@ -159,6 +161,46 @@ var _ = SIGDescribe("PodTemplates", func() {
 
 		err = wait.PollImmediate(podTemplateRetryPeriod, podTemplateRetryTimeout, checkPodTemplateListQuantity(f, "podtemplate-set=true", 0))
 		framework.ExpectNoError(err, "failed to count required pod templates")
+
+	})
+
+	ginkgo.It("should replace a pod template", func() {
+
+		ptClient := f.ClientSet.CoreV1().PodTemplates(f.Namespace.Name)
+		ptName := "podtemplate-" + utilrand.String(5)
+
+		ginkgo.By("Create a pod template")
+		ptResource, err := ptClient.Create(context.TODO(), &v1.PodTemplate{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:   ptName,
+				Labels: map[string]string{"podtemplate-set": "true"},
```

Comment:
Copy/paste from the previous test? - should probably be removed here because we don't use it in the test.

Reply:
Fixed. Removed Line 176 reference for `Labels`